### ISSUE TITLE
Error when trying to npm install version 1.4.0

### ISSUE THREAD

**Issue by ericlewis** [sanack/node-jq#154]
I think it is related to this commit that was added: https://github.com/sanack/node-jq/commit/f6e25bc98941cc0f23504f5d38f7fdebecf90690#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

```bash
npm WARN deprecated gulp-util@3.0.8: gulp-util is deprecated - replace it, following the guidelines at https://medium.com/gulpjs/gulp-util-ca3b1f9f9ac5
npm ERR! path /Users/ericlewis/.npm-packages/lib/node_modules/node-jq/bin/jq
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/Users/ericlewis/.npm-packages/lib/node_modules/node-jq/bin/jq'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ericlewis/.npm/_logs/2019-01-04T18_09_38_495Z-debug.log
macbook:rosetta ericlewis$ 
```

**Comment by ericlewis** [comment]
this happens regardless of -g or local.

**Comment by kaylah** [comment]
I've experienced the same problem installing 1.4.0

**Comment by agosto-thomasbohmbach** [comment]
Same here. Reverted back to `v1.3.1`

**Comment by ericlewis** [comment]
You can try @trycom/node-jq for 1.4 sans bin issue

**Comment by davesnx** [comment]
Published a new version 1.5.0 without the issue.

Thanks for reporting, I didn't know that npm was checking for binaries and throw if they weren't there before post install time.
